src/main/scala/org/broadinstitute/dsde/rawls/model/StatisticsModel.scala

```diff
@@ -0,0 +1,29 @@
+package org.broadinstitute.dsde.rawls.model
+
+import spray.json._
+
+/**
+ * Created by mbemis on 7/20/16.
+ */
+
+sealed trait Statistic
+case class SingleStatistic(value: Double) extends Statistic
+case class SummaryStatistics(min: Double, max: Double, mean: Double, stddev: Double) extends Statistic
+case class StatisticsReport(startDate: String, endDate: String, statistics: Map[String, Statistic])
+
+object StatisticsJsonSupport extends JsonSupport {
+  implicit val SingleStatisticFormat = jsonFormat1(SingleStatistic)
+  implicit val SummaryStatisticsFormat = jsonFormat4(SummaryStatistics)
+
+  implicit object StatisticFormat extends RootJsonFormat[Statistic] {
+    override def write(obj: Statistic): JsValue = obj match {
+      case SingleStatistic(value) => JsObject(Map("value" -> JsNumber(value)))
+      case SummaryStatistics(min, max, mean, stddev) => JsObject(Map("min" -> JsNumber(min), "max" -> JsNumber(max), "mean" -> JsNumber(mean), "stddev" -> JsNumber(stddev)))
+    }
+    override def read(json: JsValue): Statistic = json match {
```

Comment:
In that case, can you make it even more useless?  Remove the match.  Maybe even make it explicitly unimplemented with the special `???` operator, like `override def read(json: JsValue) = ???` if the compiler will allow it.